### PR TITLE
[feat] Add tasks for time management

### PR DESCRIPTION
**Pull request by Hamdrive**
## Overview

Features:
1. A separate tasks page which shows all pending tasks (if any)
2.  Ability to add or remove existing tasks
3.  New modal component to add information about the task

## Deployment

https://drivers-pomodoro-git-tasks-hamdrive.vercel.app/tasks

## Preview
image
image
image

### CODE REVIEW

**Review by shivampandey0 — APPROVED** [review]
The code looks good and works as expected. More optimizations could be done later.
Good to merge

File: src/components/Modal/Modal.jsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  );
+
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+    setInfo((prevInfo) => ({ ...prevInfo, [name]: value }));
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log(info);
```
Comment (Tanay-J) [inline src/components/Modal/Modal.jsx]: Can be removed

File: src/components/Modal/Modal.module.css [hunk 2]
```diff
@@ -0,0 +1,72 @@
+.overlay {
+  width: 100%;
+  height: 100%;
+  left: 0;
+  top: 0;
+  right: 0;
+  bottom: 0;
```
Comment (Tanay-J) [inline src/components/Modal/Modal.module.css:7]: You may use shorthand ```inset``` property instead

**Review by Tanay-J — APPROVED** [review]
Good work! Self explanatory naming scheme and good practices like destructuring make the code very readable.
Approving the changes.

File: src/components/Modal/Modal.jsx (hunk 1 above)
Comment (Hamdrive) [reply src/components/Modal/Modal.jsx]: Yes thanks for catching that, removed!

File: src/components/Modal/Modal.module.css (hunk 2 above)
Comment (Hamdrive) [reply src/components/Modal/Modal.module.css:7]: Thanks for showing this CSS property, shorthand handy!
But for readability I will prefer to leave it as is.

**Comment by Hamdrive** [comment]
PR successfully reviewed by 2 team members, Changes recommended and implemented and no issues found.
Thus PR can be merged to dev and closed

**Pull request merged**